src/main/frontend/handler/editor.cljs

```diff
@@ -3116,6 +3116,18 @@
     (let [input (gdom/getElement id)]
       (close-autocomplete-if-outside input))))
 
+(defn editor-on-height-change!
+  [id]
+  (fn [row-height]
+    (let [input (gdom/getElement id)
+          top (gobj/get (.getBoundingClientRect input) "top")
+          cursor-y (+ top row-height)
+          vw-height (.-height js/window.visualViewport)]
+      (when (<  vw-height (+ cursor-y 40))
```

Comment:
`40` is the height of toolbar on mobile

Reply:
Better add a comment or global const for magic values.